minos/common/model/abc/dto_model.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+from __future__ import (
+    annotations,
+)
+
+import typing as t
+
+from ...protocol import (
+    MinosAvroProtocol,
+)
+from .fields import (
+    ModelField,
+)
+from .model import (
+    MinosModel,
+)
+
+
+class DtoModel(MinosModel):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
It's better not to define the `init` class if not any new attributes has been set.

Reply:
Right, done !!!